### ISSUE TITLE
ClojureScript plus CanvasKit = Browser target?

### ISSUE THREAD

**Issue by nedtwigg** [HumbleUI/HumbleUI#20]
I have been perusing the next-gen UI toolkit domain for a while, and I think the two most interesting efforts are this project and [Makepad](https://github.com/makepad/makepad).

Makepad is targeting desktop as well as the browser. The browser is a somewhat degraded experience, but plenty good enough to be useful for demos.

Do you think HumbleUI has any chance of supporting the browser as a target? At first glance, it looks like Skija+JWM can be replaced with [CanvasKit](https://skia.org/docs/user/modules/canvaskit/) getting called through ClojureScript...

**Comment by tonsky** [comment]
Yeah, I think there’s a possibility, although I don’t really have it as a goal at this point
